CookBookProject/IOSCookBook/IOSCookBook/Scenes/Main/MainViewController.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  MainViewController.swift
+
+import UIKit
+import Foundation
+import RxSwift
+import RxCocoa
+import Then
+import NSObject_Rx
+import MGArchitecture
+import Reusable
+
+class MainViewController: UIViewController, BindableType {
+    @IBOutlet weak var tableView: UITableView!
+    
+    var viewModel: MainViewModel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+    
+    func bindViewModel() {
+        let input = MainViewModel.Input(
+            loadTrigger: Driver.just(()),
+            selectTrigger: tableView.rx.itemSelected.asDriver()
+        )
+        let output = viewModel.transform(input)
+        output.repos.asObservable()
+            .subscribe(onNext: { (arrayCategory) in
```

Comment:
```suggestion
            .drive()
            .disposed(by: rx.disposeBag)
```
ko subscribe như thế này